workbench-core/accounts/src/services/accountService.test.ts

```diff
@@ -195,7 +202,10 @@ describe('AccountService', () => {
     const response = await accountService.createOrUpdate(accountMetadata);
 
     // CHECK
-    expect(response).toEqual({ ...accountMetadata, id: 'sampleAccId' });
+    expect(response).toEqual({
+      ...accountMetadata,
+      id: `${resourceTypeToKey.account.toLowerCase()}-sampleAccId`
```

Comment:
I'm seeing `resourceTypeToKey.account.toLowerCase()` duplicated a lot when constructing the id. would it make sense to do something like `idWithUUID(resourceTypeToKey.account, uuid)` defined in an appropriate location so that we don't have the formatting logic spread out across various files?

Reply:
updated to define this once at the top of the test.